src/pages/workspaces/workspace/Workflows.js

```diff
@@ -89,6 +89,7 @@ const sortOptions = [
 
 const WorkflowCard = Utils.memoWithName('WorkflowCard', ({ listView, name, namespace, config, onExport, onCopy, onDelete, workspace }) => {
   const { namespace: workflowNamespace, name: workflowName, methodRepoMethod: { sourceRepo, methodVersion } } = config
+  const sourceRepoName = sourceRepo === 'agora' ? 'Terra' : sourceRepo
```

Comment:
Nit (existing, so feel free to ignore): We should probably have consistent capitalization on this, either `Agora` instead of `agora` or `terra` instead of `Terra`

Reply:
This is just the human-readable name. Maybe use `Utils.normalizeLabel()`?